Automattic/jetpack
# Update/add my jepack card loaded event

## Proposed changes:

* This diff adds a "loaded" event for the product cards on the My Jetpack page

## Jetpack product discussion


panfyZ-34C-p2#comment-8569

## Does this pull request change what data or activity we track or use?


Yes, this PR tracks an event when each product card loads on the My Jetpack page.

## Testing instructions:






* With this PR applied, navigate to My Jetpack
* Observe your network panel, you can filter requests by `t.gif`. 
* Make sure you see a `jetpack_myjetpack_product_card_loaded` event for each product card on the page
